### DISCUSSION TITLE
Copilot is Amazing thank you so much <3

### DISCUSSION THREAD

**Discussion by Hardmode010** [community/community#13384] [Copilot Conversations]
Copilot is my new friend

**Comment by Sergi0Martin** [comment]
Yes it is awesome, great job! I follow this steps and I added the extension with no problems.

https://github.blog/2022-03-29-github-copilot-now-available-for-visual-studio-2022/

**Reply by CptNope** [reply]
CoPilot has helped me so much I don't know who to thank for giving me access!!
